frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -7,6 +7,7 @@ import { Highlighter, Menu, MenuItem } from 'react-bootstrap-typeahead';
 import { Label } from '../Label';
```

Comment:
@jdtoombs please review these changes, I've updated the components that use ParentSelect to handle the change from returning an object to returning an id by default.